backend/question-service/controllers/question.controller.js

```diff
@@ -0,0 +1,117 @@
+const db = require('../models');
+const Question = db.questions;
+
+exports.create = (req, res) => {
+  if (!req.body.questionId
+    || !req.body.questionTitle
+    || !req.body.questionDescription
```

Comment:
Perhaps you can use the condition `req.body.constructor === Object && Object.keys(req.body).length === 0` instead and let the db request fail so that it sends an error (provided these fields have a not null constraint in the db)? Frontend will also handle input validation.

Reply:
Not too sure about constraints on the db, will have to consult with @YZTangent about the exact details. The return statement is there to deal with an issue I noticed with requests following an invalid one raising issues with editing a header or something, I can't quite recall the exact message, do let me know your thoughts